PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -558,6 +564,16 @@ def miniAOD_customizeMC(process):
     process.patJets.embedGenPartonMatch = False
     #also jet flavour must be switched
     process.patJetFlavourAssociation.rParam = 0.4
+    
+    from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+    from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+    (pp_on_AA_2018 | pp_on_PbPb_run3).toModify(process.slimmedGenJetsAK8, cut = 'pt>9999', nLoose = 0)
+    (pp_on_AA_2018 | pp_on_PbPb_run3).toModify(process.slimmedGenJetsAK8SoftDropSubJets, cut = 'pt>9999', nLoose = 0)
+    from PhysicsTools.PatAlgos.producersHeavyIons.heavyIonJetSetup import aliasCsJets, removeJECsForMC, removeL1FastJetJECs
+    (pp_on_AA_2018 | pp_on_PbPb_run3).toModify(process, func = lambda proc: aliasCsJets(proc, 'AKCs4PF'))
```

Comment:
as noted in PhysicsTools/PatAlgos/python/producersHeavyIons/heavyIonJetSetup.py
I prefer some other solution to the aliasing